docs/src/components/TopBanner/TopBanner.js

```diff
@@ -17,17 +17,36 @@
  */
 /* @flow strict */
 
-import React from 'react';
+import React, { type Node } from 'react';
 import { cssModules } from 'bpk-react-utils';
+import BpkLink from 'bpk-component-link';
 
 import STYLES from './TopBanner.scss';
 
 const getClassName = cssModules(STYLES);
 
-const TopBanner = (props: { className: ?string }) => {
-  const { className, ...rest } = props;
+type Props = {
+  children: Node,
+  className: ?string,
+  links: Array<{ title: string, href: string }>,
+};
+
+const TopBanner = (props: Props) => {
+  const { children, className, links, ...rest } = props;
   return (
-    <div {...rest} className={getClassName('bpkdocs-top-banner', className)} />
+    <div {...rest} className={getClassName('bpkdocs-top-banner', className)}>
+      {children}
```

Comment:
If we're passing links in as an object, do we need `children`?

Reply:
`children` is used for the rest of the text — in this case it says "Pull request build"